.circleci/config.yml

```diff
@@ -75,3 +75,41 @@ jobs:
 
       - store_artifacts:
           path: doc/build/html
+
+  image:
```

Comment:
Maybe (eventually) a more descriptive name for the run e.g. `image-comparison` or `pytest-mpl`. Definitely don't need to worry about it now though.

Reply:
Will do.  I ended up with the shorter title during debugging when I was struggling to get this working (and wasn't sure what the problem was).